CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Support stored procedure register with packages given as Python modules.
 - Added support for structured type schema parsing.
+- Removed import of modin.pandas.Index referring to native pandas index, instead having it import our own class that is currently a wrapper to native pandas index
```

Comment:
When you say "our own class" do you still mean `modin.pandas.Index`? If yes, it would make more sense to say that the new class is now a wrapper class instead of the reference class behavior in previous Snowpark pandas versions.

Reply:
well the `modin.pandas.Index` currently just maps to a native pandas index, we are creating a Index class and changing what it refers to